src/codewhisperer/service/transformByQHandler.ts

```diff
@@ -560,7 +560,7 @@ export async function pollTransformationJob(jobId: string, validStates: string[]
 
 async function checkIfGradle(projectPath: string) {
     const gradleBuildFile = await vscode.workspace.findFiles(
-        new vscode.RelativePattern(projectPath, '**/build.gradle'),
+        new vscode.RelativePattern(projectPath, 'build.gradle'),
```

Comment:
we only want to recognize a project as being gradle if we see the build.gradle in the root directory

Reply:
is `findFiles` still needed then?

Reply:
No, `fs.existsSync()` is an acceptable alternative, will make the change now.